python/sglang/srt/utils.py

```diff
@@ -1425,6 +1425,38 @@ def is_valid_ipv6_address(address: str) -> bool:
         return False
 
 
+def configure_ipv6(server_args):
```

Comment:
```suggestion
def configure_ipv6(dist_init_addr):
```

Do not accept `server_args` as input. Only takes what is really needed